### DISCUSSION TITLE
Patreon Supporters

### DISCUSSION THREAD

**Discussion by bcholmes** [bcholmes/StarTrek2d20#236] [Announcements]
Hey folks,

I'm thinking of opening a Patreon for people who want to support the ongoing development of the character creator.

I first started maintaining this site over three years ago, and ever since I started, I've poo-poo'ed the idea of Patreon or LibrePay. I've always supported this site as a labour of love; that's not going to change. So why am I changing my opinion of crowdfunding, now? There are a few reasons. 

I've enjoyed interacting with folks about what they'd love to see in the site, but the number of requests for new features has now exceeded my ability to handle them all in a timely fashion. I think that one of the things that a community of supporters can help with is providing a better way to engage with folks about, say, weighing in on priorities. 

It's also true that I tend to spend my time working on things that I particularly enjoy. There's a concept in open source programming: that often folks tend to try to solve problems that scratch their own personal itches: I'm often guilty of that. The best open source projects also help others scratch their own itches. I tend to spend a certain amount of time building the things that interest me. To date, I've been happy that other folks have found value in the things I've added: things like the token generator and the like. 

In some (admittedely-infrequent) cases, I've tried to run polls to help guide my efforts. Sadly, it's hard to find the best place to conduct those polls. The Modiphius official Discord channel doesn't easily support polls. And no other social media platform has a good concentration of STA players. Facebook is the least-worst place, at the moment. But if I'm honest, I don't particularly love Facebook.

From what I know of Patreon, it can give me a place where I can interact with a community of committed fans and take guidance about what people are really hungry for.

The other uncomfortable topic is money. The very words "support me on Patreon" generally sound like "give me your money!" To my Star Trek-influenced ears (eagerly awaiting that post-scarcity economy), that ask sits uneasily with me.

But there are some costs that I've started to incur in support of building this tool. I've always had my own server (the bcholmes.org URL) that I pay for. The character creator is now the largest contributor to traffic on that server. Those aren't huge costs, but they are costs. 

I additionally periodically purchase 3D models of some of the rarer starship classes to help me produce the outlines that appear on the starship sheets. I also commission art to help expand the token generator. 

So there have been small costs here and there, that a Patreon could offset. But, I want to be clear that I'm not trying to make a living off of any supporters. This is still a labour of love.

Here are some things that I'm not considering. First, I'm not interested in a "paid tier" of features. I think that part of what makes this site powerful is that it's easy and free for everyone. I don't want to change that. If you can't affort to support a Patreon or if there's some other reason why you can't or won't use it, that shouldn't preclude you from using the site. Maybe there's an option for an advanced preview of new features; I'd have to think through how easy and/or useful that'd be. But no paid-tier-only features, ever.

Second: I never really want to be the sort of person who is constantly clinking a tin cup looking for support. If supporting a Patreon is something that you feel good about, then great. If not, that's okay, too. I never want to turn away asks for new features just because they don't come from paying supporters.

Perhaps your response to all this is to ask, "but what do I get out of supporting a Patreon?" I *think* the answer is "more ability to influence future features" and "the knowledge that support is turning in to more commissions of token art to more quickly flesh out tokens (and other such features)." Maybe that's not enough to entice you: I'm certainly open to additional ideas, here.

This isn't a final decision. I'm still weighing if this is a good idea. But if you have thoughts, please weigh in.

**Comment by pdpatterson** [comment]
I would love to support you via Patreon!

**Comment by mer0mingian** [comment]
Same here. You have - for years - provided the go-to solution to build chars for STA, which a lot of people are very thankful for! I deeply resonate with your reasoning and currently my only regret is lacking the expertise to support you with PRs instead of donations, but if the latter is a good for you... code long (but in small commits) and prosper!

**Comment by BarthSarafin** [comment]
I can only agree with all the others. I think you do a marvel for the STA community and your work is my basis for all my STA needs.

**Comment by njhdunn** [comment]
I would enjoy supporting you via Patreon as well. This is the kind of project that just feels right to support - it facilitates fun, you are providing the source code to the public, and you won't be gating features behind a paywall. Let us help offset those hosting costs, at least.

**Comment by echo-aj** [comment]
I've made use of the character creator for quite a while. (Maybe since early days? Who knows in these post-pandini times.) And I've used other equivalent tools for other games with no issue spending a dollar or two to help support the creators. Your efforts have helped reduce the amount of time needed to prep for games, and have enabled me to noodle around making characters just for the fun of it. That's worth at least a couple of bucks for services already rendered, and if it means there's a level of stability going forward when new content is released for the game or an incentive to work on the less exciting-to-develop features that might be requested then so much the better.

**Comment by virgil38** [comment]
I'd support you.

**Comment by servingworlds** [comment]
I will support you wherever you hang your crowdfunding hat BUT I strongly recommend you do NOT sign up with ANY platform. 

Patreon is hit by scandals and changes to services and pricing every few years like clockwork. Many creators use third party tools to host their own patreon-like functionality on a public website THEY CONTROL.

These platforms let you attract new audiences, but you have your own dedicated followers already. Any platform will take a percentage from having you, instead of you keeping that money after doing a little work yourself. And if they burn out and fade away, they take their audiences down with them.

My two cents...and thank you again for all you do!

**Comment by greatkithain** [comment]
I will support you, mate. Great things as your job for STA should be rewarded in any form. Please, co sider other alternatives to Patreon. These intermediate platforms takes too much part of the donations and many creators are moving to another platform.

**Comment by Dogsrule333** [comment]
May I suggest you also look at Ko-fi.com  I believe you may find they tend to take less then Patreon.  Not sure if they will have everything you need, but something to check out.

**Comment by MasterofMankind** [comment]
I would most definitely support a Patreon. Your efforts make STA unique and represent an indispensible aid to facilitate play. I wont get into any controversy regarding the crowdfunding platforms, but I strongly suggest you adhere to Patreon if only to demonstrate your efforts as being above-board. All it takes is someone suspicious at Paramount to not only sink the effort, but the app altogether. That being said, Paramount may take a dim view of this no matter how good your intentions are. I would support any efforts to support you.

**Comment by thecamerageek** [comment]
I would support you. This tool is one of the best things to promote Star Trek Adventures there is. It only seems fair to me that I slide you a little cash for all of the joy you have helped me share with my players.

**Comment by matthew81** [comment]
I would support this, I have used this site since you first started it and It is a great resource. I think my time as an online GM with players around the planet would have been so much harder if not for the tools you have created. I know you said its not a money thing but you have put so much time and effort into this that you should get something for that.

**Comment by Felderburg** [comment]
Did this happen? I've been busy with IRL stuff for a while, and would definitely support a patreon.

Edit: Yes it did,
